src/components/Display.js

```diff
@@ -1,9 +1,10 @@
 import PropTypes from 'prop-types';
 
-const Display = (props = 'O') => <h1>{props.result}</h1>;
+const Display = (props) => <h2 className ='input-value'>{props.result || props.next || '0'}</h2>;
```

Comment:
## Optional 
- [ ] The UI isn't great when it comes to the displaying of what's to get rendered.
While we do have our function store and the next value typed, the characters never get displayed, and the user never gets to see what they keyed; a bad User Experience. Would you mind changing it up a bit (in terms of the display), a user to see the digits (and, hopefully, operations that get typed for great User experience?  This is what we currently have 👇 
![calculator](https://user-images.githubusercontent.com/22345961/101994900-287f5200-3cd7-11eb-9197-b81c8d08890b.gif)